pom.xml

```diff
@@ -655,6 +676,36 @@
                 <version>5.2</version>
             </dependency>
             <dependency>
+                <groupId>org.codehaus.jackson</groupId>
+                <artifactId>jackson-core-asl</artifactId>
+                <version>1.9.13</version>
+            </dependency>
+            <dependency>
+                <groupId>org.codehaus.jackson</groupId>
+                <artifactId>jackson-mapper-asl</artifactId>
```

Comment:
Specifying the version in dependencyManagement. There are conflicts in subprojects, where different dependencies want different version of this lib.